01week/datatypes.js

```diff
@@ -0,0 +1,55 @@
+console.log(Date())
+
+
+
+// #3 Write a JavaScript program to convert a string to the number
+const numberToString = (str) => {
+  return parseInt(str)
+}
+
+console.log(numberToString("9"))
+
+// Write a JavaScript program to display the current day and time.
+console.log(Date())
+// Write a JavaScript program to convert a number to a string.
+var num = 77 
+num.toString()
+// Write a JavaScript program to convert a string to the number.
+Number('55556666')
+// Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+// Boolean
+console.log(typeof (10 > 9))
+// Null
+console.log(typeof (null))
+// Undefined
+console.log(typeof (undefined))
+// Number
+console.log(typeof (10))
+// NaN
+// console.log(typeof ( ))
```

Comment:
Think about doing a math function that adds a number and a variable. The program wouldn't know what to respond with because it expects a number for both parameters. It would return Not a Number (NaN). What are some possible functions you could write to return NaN?